crates/services/consensus_module/poa/src/service_test/test_time.rs

```diff
@@ -0,0 +1,81 @@
+use std::time::Duration;
+
+use fuel_core_types::tai64::{
+    Tai64,
+    Tai64N,
+};
+use tokio::{
+    sync::watch,
+    time::Instant,
+};
+
+use crate::ports::GetTime;
+
+pub struct TestTime {
```

Comment:
How do you feel about naming this `TokioTime`?

Reply:
I appreciate the name suggestion, but I don't think it is appropriate here because while this struct tracks tokio time, it can still be advanced and rewinded independently of the tokio time, and the tokio time may be advanced without this struct following it.

The purpose of observing tokio time in this struct is to allow this time to follow the tokio time, but it cannot control it unfortunately due to both the auto-advancing feature of tokio and the availability of any other function to advance or rewind the tokio time.